frontend/packages/ceph-storage-plugin/src/utils/modal-common.ts

```diff
@@ -0,0 +1,7 @@
+import { StorageClusterKind } from '../types';
```

Comment:
We can change the filename to `feature-common.ts` maybe.

Reply:
ack